languages/kotlin/exercises/concept/functions-basic/src/main/kotlin/FunctionsBasic.kt

```diff
@@ -0,0 +1,3 @@
+fun increment(x: Int, amount: Int = 1): Int {
```

Comment:
Just as a thought: would it make sense for a Concept Exercise about functions to have the students define the functions from scratch? And then maybe have different functions, like one without parameters and one with parameters?

Reply:
> Just as a thought: would it make sense for a Concept Exercise about functions to have the students define the functions from scratch?

Considering that we have hints and references to documentation, it might be a good idea.

> And then maybe have different functions, like one without parameters and one with parameters?

I guess, there is need to teach "functions without arguments" at this point as we are assuming that students are familiar with any other language. The point of this exercises it to serve as one of exercises to introduce some basic "shape" of the language so we can build other exercises. For example, integer numbers are used here as well, but I think it might be better to introduce them properly in a different exercise.

Reply:
> For example, integer numbers are used here as well, but I think it might be better to introduce them properly in a different exercise.

I think so too. Integers probably have enough interesting properties to warrant a separate exercise.